src/libs/E2E/actions/e2eLogin.ts

```diff
@@ -1,16 +1,36 @@
+/* eslint-disable rulesdir/prefer-actions-set-data */
+
 /* eslint-disable rulesdir/prefer-onyx-connect-in-libs */
+import Config from 'react-native-config';
 import Onyx from 'react-native-onyx';
-import * as Session from '@userActions/Session';
+import {Authenticate} from '@libs/Authentication';
+import CONFIG from '@src/CONFIG';
 import ONYXKEYS from '@src/ONYXKEYS';
 
+function getConfigValueOrThrow(key: string): string {
+    const value = Config[key];
+    if (value == null) {
+        throw new Error(`Missing config value for ${key}`);
+    }
+    return value;
+}
+
+const e2eUserCredentials = {
+    email: getConfigValueOrThrow('EXPENSIFY_PARTNER_PASSWORD_EMAIL'),
+    partnerUserID: getConfigValueOrThrow('EXPENSIFY_PARTNER_USER_ID'),
+    partnerUserSecret: getConfigValueOrThrow('EXPENSIFY_PARTNER_USER_SECRET'),
+    partnerName: CONFIG.EXPENSIFY.PARTNER_NAME,
```

Comment:
Why no `getConfigValueOrThrow` here and bellow?

Reply:
Because the values are already defined in the `CONFIG` with the exact same fallbacks we need for the e2e testing account, so I thought to just reuse the value from there 🤔